ansible_collections/arista/avd/plugins/action/yaml_templates_to_facts.py

```diff
@@ -8,6 +8,8 @@
 from ansible.plugins.filter.core import combine
 from ansible.plugins.loader import lookup_loader
 from ansible_collections.arista.avd.plugins.module_utils.strip_empties import strip_null_from_data
+from datetime import datetime
+import yaml
 
 
 class ActionModule(ActionBase):
```

Comment:
In line 16, is it ok to set ```task_vars = {}``` instead of line 17,18

Reply:
No, because if the module is called with the argument `task_vars=None` it would not take the default value. So this is a way of ensuring that both no value and the `None` value will lead to an empty dict.